src/main/java/com/microsoft/azure/functions/annotation/FixedDelayRetry.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>
+ * Defines a retry strategy where a fixed delay is used between retries.</p>
+ *
+ * @since 1.0.0
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface FixedDelayRetry {
+    /**
+     * The maximum number of retries that will be attempted.
+     * @return The delay between retries.
+     */
```

Comment:
Introduce Enum RetryStrategy like you  had in previous commit. Set the RetryStrategy to "FixedDelay" --> this mean mvn plug just have to convert information in the annotation directly to function.json

Reply:
Hi @pragnagopa  I'm little confuse. Once we separate the annotation, why I need the Strategy? 

![image](https://user-images.githubusercontent.com/1390976/99306199-b0be2480-2809-11eb-8118-79d6f6849dc5.png)

Reply:
I imagine a generic implementation in mvn plug in to convert this retry annotation to `retry` section in `function.json` --> If there is an enum then mvn plug in can directly convert this annotation to `retry` section. This will allow less changes if library introduces a new retry strategy then mvn does not have to be updated. "Nide to have" does not have to block in this :)

Reply:
Sure @pragnagopa  then, how we can avoid the validation logic on the maven plugin side?

Reply:
I'm ok for rollback the change and make the annotation into one again.

Reply:
Sorry. I think there is mis-understanding. Two annotations OK! Just introduce new property RetryStrategy and set it so that mvn plugin can use the value directly.

Reply:
Hi @pragnagopa 
I see what you mean!  I update it.  Looks good?

Reply:
Done